CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ Versioning](https://semver.org/spec/v2.0.0.html) for `libnopegl`.
 
 ### Removed
 - `ResourceProps.variadic` bool flag as it was never a functional interface
+- `NGL_CAP_NPOT_TEXTURE` as we mandate support for npot textures
```

Comment:
can you explicit "non-power of two dimensions for textures" instead of "npot textures"?

Reply:
Adjusted commit desc and changelog.